packages/expo-cli/src/commands/init.ts

```diff
@@ -161,53 +163,194 @@ async function action(projectDir: string, command: Command) {
     packageManager = 'npm';
   } else if (await shouldUseYarnAsync()) {
     packageManager = 'yarn';
-    log('Using Yarn to install packages. You can pass --npm to use npm instead.');
+    log.newLine();
+    log('🧶 Using Yarn to install packages. You can pass --npm to use npm instead.');
+    log.newLine();
   } else {
     packageManager = 'npm';
-    log('Using npm to install packages. You can pass --yarn to use Yarn instead.');
+    log.newLine();
+    log('📦 Using npm to install packages. You can pass --yarn to use Yarn instead.');
+    log.newLine();
   }
 
-  let projectPath = await Exp.extractAndInitializeTemplateApp(
-    templateSpec,
-    path.join(
-      parentDir,
-      dirName || ('expo' in initialConfig ? initialConfig.expo.slug : initialConfig.name)
-    ),
-    packageManager,
-    initialConfig
-  );
+  let extractTemplateStep = logNewSection('Downloading and extracting project files.');
+  let projectPath;
+  try {
+    projectPath = await Exp.extractAndPrepareTemplateAppAsync(
+      templateSpec,
+      path.join(
+        parentDir,
+        dirName || ('expo' in initialConfig ? initialConfig.expo.slug : initialConfig.name)
+      ),
+      initialConfig
+    );
+    extractTemplateStep.succeed('Downloaded and extracted project files.');
+  } catch (e) {
+    extractTemplateStep.fail(
+      'Something went wrong in downloading and extracting the project files.'
+    );
+    throw e;
+  }
+
+  // for now, we will just init a git repo if they have git installed and the
+  // project is not inside an existing git tree, and do it silently. we should
+  // at some point check if git is installed and actually bail out if not, because
+  // npm install will fail with a confusing error if so.
```

Comment:
```ts
async function isGitInstalledAsync(): Promise<boolean> {
    try {
      await spawnAsync('git', ['--version']);
      return true;
    } catch {
      return false;
    }
  }
```

Reply:
yeah let's do it in a followup pr - i want to make sure we have a good message around it and provide nice installation instructions. related issue https://github.com/expo/expo-cli/issues/1725